src/components/docs/Excerpt/styles.module.css

```diff
@@ -50,6 +55,6 @@
   }
 }
 
-html[data-theme="dark"] .root {
+html[data-theme="dark"] .root p {
```

Comment:
This is not correct. The color variable is used in root, so if you change it on the child p it won't do anything

Reply:
Nope, this works as expected